stackguardian_connector_cloud/variables.tf

```
@@ -23,42 +25,47 @@ variable "org_name" {
 
 variable "aws_access_key_id" {
   type = string
-  description = "AWS ACCESS Key ID"
-  default = ""
+  description = "your AWS acoount access key"
 }
+
 variable "aws_secret_access_key" {
   type = string
-  description = "AWS ACCESS Key Secret"
-  default = ""
+  description = "your AWS account secret access key"
 }
+
 variable "aws_default_region" {
   type = string
-  description = "AWS Default Region for Connector"
-  default = ""
+  description = "any default region you want to set, for all your deployments"
 }
 
 ################
 # AZURE_STATIC Credentials
 ################
 
-
 variable "armTenantId" {
   type = string
-  description = "Azure Tenant ID"
-  default = ""
+  description = "your azure account tenant id"
 }
+
 variable "armSubscriptionId" {
   type = string
-  description = "Subscription ID"
-  default = ""
+  description = "your azure subscription id"
 }
+
 variable "armClientId" {
   type = string
-  description = "Client ID for Enterprise App"
-  default = ""
+  description = "your azure client id"
 }
+
 variable "armClientSecret" {
   type = string
-  description = "Client Secret for Enterprise App"
-  default = ""
+  description = "your azure client secret id"
```

It is the client secret NOT the client secret id
```suggestion
  description = "your azure client secret"
```